src/MultiplayerMod/MultiplayerMod.csproj

```diff
@@ -61,4 +61,7 @@
     <ItemGroup>
         <ProjectReference Include="..\MultiplayerMod.AttributeProcessor\MultiplayerMod.AttributeProcessor.csproj" Private="false"/>
     </ItemGroup>
+    <ItemGroup>
+      <PackageReference Include="AssemblyExposer.MSBuild.Task" Version="0.1.3" Private="true"/>
```

Comment:
Different indentation, could you please check if it's possible to configure it via `.editorconfig`? It has `indent_size = 4`, but seems like there's no effect on `.csproj` files?